susemanager-utils/testing/automation/update-obs-environment-cr-project.sh

```diff
@@ -1,15 +1,93 @@
 #!/bin/bash
 
-if [ $# -ne 1 ];then
-    echo "usage: $0 N"
+parent_project="systemsmanagement:Uyuni:Master"
+
+usage_and_exit() {
+    echo "usage: $0 N [test_project]"
     echo "where N is the environment"
-    echo "$0 will add new packages in systemsmanagement:Uyuni:Master to systemsmanagement:Uyuni:Master:TEST:N:CR"
+    echo "and project is the test_project, which by default is ${parent_project}:N:CR" 
+    echo "$0 will add new packages from ${parent_project} to test_project"
+    exit -1
+}
+
+update_project() {
+    tproject=${1}
+    pproject=${2}
+    rname=${3}
+    echo "Updating ${tproject} from ${pproject} with repo ${rname}"
+    osc ls ${tproject} > /dev/null
```

Comment:
Since you are managing the error at the if, I guess you don't need `stderr`? 
```suggestion
    osc ls ${tproject} &> /dev/null
```